### ISSUE TITLE
Automated install only works with default username

### ISSUE THREAD

**Issue by TubularCorpration** [Audio-Injector/Octo#52]
Just a heads up, the "error exit status 2" during the install that a few people have reported is happening because the package tries to locate .asoundrc in /home/pi/ and will not work when logged in with any other username.

I am using mine for a mostly-offline project with a fresh Raspberry Pi OS 64 bit, so it was easiest for me to reinstall the OS and use the default username and that seems to have worked, but it's not a good option for most people so it would be great to make the automatic install package detect or prompt for the correct home directory instead of assuming the default and needing manual installation otherwise.

**Comment by toaniel** [comment]
This is correct. I had to reinstall the OS with user pi for it to work.

**Comment by madrulane** [comment]
Yes, it irked me too... But you can modify the install after the fact (or manual install).
Also the installer script is not going to make it work anyway, as it has issues w. the new kernel (need to downgrade kernel).
